app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :likes_counter
+
+  def update_like_counter
```

Comment:
- [ OPTIONAL ] It would be better if use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_like_counter` method every time a new like has been saved to the database, then make the `update_like_counter` method private. :+1: